tokens/src/components/Modal.json

```diff
@@ -13,6 +13,7 @@
       }
     },
     "content": {
+      "color": { "value": "null", "source": "$modal-content-color" },
```

Comment:
please check that variables in this file are present in `_variables.scss` file in `Modal` component, I think some of them were deleted in one of the previous commits.